docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
If we have username and ssh types I should be able to easily create them.
Those aren't exclusive to source, so we shouldn't be pretending like they
are.

On Wed, Sep 9, 2015 at 10:21 AM, Ben Parees <notifications@github.com>
wrote:

> In docs/generated/oc_by_example_content.adoc
> <https://github.com/openshift/origin/pull/4433#discussion_r39048007>:
>
> > @@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
> >  ====
> >
> >
> > +== oc secrets new-sourcecfg
> > +Create a new source secret
>
> (I don't really see what's wrong with the existing proposed CLI)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4433/files#r39048007>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift